test/test_transforms_tensor.py

```diff
@@ -109,109 +109,118 @@ def _test_op(func, method, device, fn_kwargs=None, meth_kwargs=None, test_exact_
         (F.equalize, T.RandomEqualize, None, {})
     ]
 )
-def test_random(func, method, device, fn_kwargs, match_kwargs):
-    _test_op(func, method, device, fn_kwargs, fn_kwargs, **match_kwargs)
+@pytest.mark.parametrize('channels', [1, 3])
+def test_random(func, method, device, channels, fn_kwargs, match_kwargs):
+    _test_op(func, method, device, channels, fn_kwargs, fn_kwargs, **match_kwargs)
 
 
 @pytest.mark.parametrize('device', cpu_and_gpu())
 class TestColorJitter:
 
     @pytest.mark.parametrize('brightness', [0.1, 0.5, 1.0, 1.34, (0.3, 0.7), [0.4, 0.5]])
-    def test_color_jitter_brightness(self, brightness, device):
+    @pytest.mark.parametrize('channels', [1, 3])
```

Comment:
Let's put channels to the class as done for `device`

Reply:
Oh, I see. It's definitely better.